src/main/java/seedu/address/logic/parser/AddMeetingCommandParser.java

```diff
@@ -19,11 +22,19 @@ public AddMeetingCommand parse(String args) throws ParseException {
         ArgumentMultimap argMultimap =
                 ArgumentTokenizer.tokenize(args, PREFIX_NAME, PREFIX_DATE, PREFIX_TIME);
```

Comment:
I think Thomas changed requirement from add by client name to add by client index 💭

Reply:
I have changed the command to accept the index based on the filtered list of clients